packages/vuetify/src/services/breakpoint/index.ts

```diff
@@ -65,7 +65,7 @@ export class Breakpoint extends Service implements IBreakpoint {
     }
     this.scrollBarWidth = (
       options.scrollBarWidth ||
```

Comment:
`this.scrollBarWidth = options.scrollBarWidth != null ? options.scrollBarWidth : this.scrollBarWidth`